src/scss/skin-modern/components/_airplaytogglebutton.scss

```diff
@@ -4,13 +4,17 @@
 .#{$prefix}-ui-airplaytogglebutton {
   @extend %ui-button;
 
-  background-image: svg('assets/skin-modern/images/airplay.svg');
+  &::before {
```

Comment:
As discussed offline, it would be nice if the images could remain as is and we insert a dummy element in `::before` for correct sizing.